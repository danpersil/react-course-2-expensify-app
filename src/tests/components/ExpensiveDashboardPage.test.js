import React from "react";
import { shallow } from "enzyme";
import ExpensiveDashboardPage from "../../components/ExpensiveDashboardPage";

test('Should render ExpenseDashboardPage corretly', () => {
    const wrapper = shallow(< ExpensiveDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});