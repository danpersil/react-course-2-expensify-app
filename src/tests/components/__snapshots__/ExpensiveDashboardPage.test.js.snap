// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashboardPage corretly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
